application-template/public/index.html

```diff
@@ -0,0 +1,32 @@
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <title>ubio application bundle</title>
+        <link href="./index.css" rel="stylesheet">
+        <script type="module">
+            import { createJob, runJob } from './index.js';
+
+            window['kick-off'].onsubmit = async evt => {
+                evt.preventDefault();
+
+                const form = new FormData(evt.target); // Not used yet.
+                const { token, jobId, serviceId, input, local } = await createJob();
```

Comment:
Should we try-catch here?

Reply:
Possibly, but since it's example code I'll defer that to later.